src/ui/src/Validation/validateCreateAuction/validateCreateAuction.ts

```diff
@@ -0,0 +1,35 @@
+import dayjs from 'dayjs';
+import auctionTypes from '../../domain/enums/SmartAuctionType';
+import { ICreateAuctionFormValues } from '../../interfaces/Forms/CreateAuction/ICreateAuctionFormValues';
+
+function validateCreateAuction(values: ICreateAuctionFormValues): any {
+  const errors: any = {};
+
+  if (!Object.values(auctionTypes).includes(values?.auctionType)) {
+    errors.auctionType = `Unavailable auction type: available types are: ${Object.keys(
+      auctionTypes
+    ).join(', ')}\n`;
+  }
+
+  if (values?.description?.length >= 200) {
```

Comment:
I think it's best to choose object destructuring in this case. This allows us use a short name without "values". And even better would be to make an interface for the error like ICreateAuctionFormErrors.
const { Auction type, description, name, start time } = values;
if (name) if (description) ...

Reply:
I'd agree with you in case this interface would be used anywhere else. I don't think it's needed in the current context because we easily can see all the fields which are being validated just in the scope of the function

Reply:
As for object destructuring, I don't think we have a really long chain calls. + the majority of validations has single check only